sds_data_manager/constructs/indexer_lambda_construct.py

```diff
@@ -105,6 +106,10 @@ def __init__(
 
         # Write science data info to db with
         # status SUCCEEDED
+        event_rule_prefixes = [
+            {"prefix": f"imap/{instrument}/"} for instrument in VALID_INSTRUMENTS
+        ]
+        event_rule_prefixes.append({"prefix": "imap/ancillary/"})
```

Comment:
Maybe call this `science_event`? I'm a bit confused though because it contains ancillary too? So this is everything but spice then instead of just science?
```suggestion
        science_event_prefixes = [
            {"prefix": f"imap/{instrument}/"} for instrument in VALID_INSTRUMENTS
        ]
        science_event_prefixes.append({"prefix": "imap/ancillary/"})
```

Reply:
yep. It's just science and ancillary event.

Reply:
Doing this filtered indexer lambda event to what it needs because we are going to add more subfolder under `imap/` and doing this helps in ignoring other events that could come in the future, such as quicklook products or cadence inputs or SPICE(which is handled by SPICE indexer lambda).